### ISSUE TITLE
Search functionality?

### ISSUE THREAD

**Issue by jaume-rodriguez** [daveho/AceGWT#30]
Hi David, 

Is there something implemented to search a specific text in the textarea (and highlight the results)?

Cheers, 
Jaume.

**Comment by ainsley** [comment]
Not in the API as of yet, but Ctrl+F keyboard shortcut should work within the editor window.

**Comment by jaume-rodriguez** [comment]
Hi Ainsley, 

Thanks for your answer. I know, I am aware of it, and would solve all my problems. But the decision is not always for me take :) 

Cheers, 
Jaume.

**Comment by jaume-rodriguez** [comment]
Hi again, 

Sorry for reopening the issue again. 
But I recently realized that the Ctrl+F functionality only search through the visible text. It doesn't detect or scroll  to the position of other occurrences of the specified text. 
Did somebody have the same issue? 

Thank you, 
Jaume.

**Comment by ainslec** [comment]
I have been using the ctrl+f functionality for years. It certainly seems to search through all text (not just visible text) for me.

**Comment by jaume-rodriguez** [comment]
Are you sure? Because it happens to me even with the original project (using Ctrl+F functionality, not their own implementation of the search):
https://ace.c9.io/build/kitchen-sink.html

What browser are you using? I've tested in both IE and Chrome with the same result. 

Thank you for your answer! 
Jaume.

**Comment by ainslec** [comment]
I've used AceGWT in many projects, and never had the problem ... The Ctrl+F implementation I'm using is just the default Ace Editor implementation, and you don't have to code or configure anything to get it to work.  I have tried this with Chrome, Firefox, IE and Edge.

I can't imagine how the problem would occur as the search function works from an internal variable that holds the document. The rendered text on the screen is not actually searched in the search function so it's a strange issue to be sure.

I'll try to put together a standalone demo to try this out in absolute isolation..
